src/router/dispatcher.rs

```diff
@@ -277,7 +279,7 @@ impl Dispatcher {
                 },
                 gateway_message = streams.next() => match gateway_message {
                     Some((gateway_stream, Ok(gateway_message))) => match gateway_stream {
-                        GatewayStream::Routing => self.handle_routing_update(&gateway_message, &shutdown, logger).await,
+                        GatewayStream::Routing => info!(logger, "ignoring routing update"),
```

Comment:
remove this by removing it from the `GatewayStream` enum and don't subscribe to routing updates